critiquebrainz/frontend/templates/macros.html

```diff
@@ -93,8 +93,19 @@
   </div>
 {%- endmacro %}
 
+{% macro show_tags(tags) %}
+  {% if tags %}
+    {% for tag in tags[: -1] %}
+      <span><a href="{{"https://musicbrainz.org/tag/%s"|format(tag['name'])}}"> {{tag['name']}}, </a></span>
+    {% endfor %}
+    <span><a href="{{"https://musicbrainz.org/tag/%s"|format(tags[-1]['name'])}}"> {{tags[-1]['name']}} </a></span>
```

Comment:
I don't understand what you are trying to do here. Why have `for tag in tags[: -1]` loop instead of `for tag in tags`?